scripts/auto-stop-idle/autostop.py

```diff
@@ -96,21 +96,26 @@ def get_notebook_name():
                 if notebook['kernel']['connections'] == 0:
                     if not is_idle(notebook['kernel']['last_activity']):
                         idle = False
+                        print('Notebook idle state is now: ', idle)
```

Comment:
Not sure if I understand why the print lines are needed. The is_idle function itself already prints out the idle state, right?